### PR TITLE
Papermill implementation for BM4 and BM8

### PR DESCRIPTION
**Pull request by wd15**
- Add notebooks for BM4
 - Make uploads more robust (less failures with broken CSV files or broken transformations, these are written to a log file)
 - update some broken data links in the BM4 meta.yaml's
 - The table of equilibrium values is missing now in this new version. I've created an issue for this to fix later, #1496.

TODO:

 - [x] ensure links go to results/ not the old comparison pages
 - [x] finish building BM8 with papermill
 - [x] ensure .gitignore is updated to not special case BM8

### CODE REVIEW

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1497.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1497.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1497.surge.sh

**Comment by wd15** [comment]
@tkphd, ready for review.

File: results/benchmark4h.1.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+benchmark_id: 4h.1
+line_plots:
+  - name: "all_data"
+    layout:
+      x_label: 'Fictive Time, <i>t</i><sub>aim</sub>&nbsp;[a.u.]'
```
Comment (tkphd) [inline results/benchmark4h.1.yaml:6]: "Fictive" feels haughty. Can we say "Time" as in BM8, or "Simulation Time", or something people won't get confusing search results for to confirm the meaning?

File: results/benchmark8a.1.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+line_plots:
+  - name: 'free_energy_1'
+    layout:
+      x_label: 'Time [a.u.]'
+      y_label: 'Free Energy, &#8497;&nbsp;[a.u.]'
+      title: '<i>r</i><sub>0</sub>=0.99<i>r</i><sup>*</sup>'
+  - name: 'solid_fraction_1'
+    layout:
+      x_label: 'Time [a.u.]'
+      y_label: 'Solid Fration, [a.u.]'
```
Comment (tkphd) [inline results/benchmark8a.1.yaml:12]:
```suggestion
      y_label: 'Solid Fraction, [a.u.]'
```

File: results/benchmark8a.1.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      title: '<i>r</i><sub>0</sub>=0.99<i>r</i><sup>*</sup>'
+  - name: 'free_energy_2'
+    layout:
+      x_label: 'Time [a.u.]'
+      y_label: 'Free Energy, &#8497;&nbsp;[a.u.]'
+      title: '<i>r</i><sub>0</sub>=<i>r</i><sup>*</sup>'
+  - name: 'solid_fraction_2'
+    layout:
+      x_label: 'Time [a.u.]'
+      y_label: 'Solid Fration, [a.u.]'
```
Comment (tkphd) [inline results/benchmark8a.1.yaml:22]:
```suggestion
      y_label: 'Solid Fraction, [a.u.]'
```

File: simulations/4h.1.html
Comment (tkphd) [inline simulations/4h.1.html:1]: This is way too many variants.

**Review by tkphd — APPROVED** [review]
Minor changes, but this looks good!

File: results/benchmark4h.1.yaml (hunk 1 above)
Comment (wd15) [reply results/benchmark4h.1.yaml:6]: Happy to change it, but that terminology is used in the paper, see fig 5 in [paper](https://www.sciencedirect.com/science/article/pii/S092702561830168X)

Comment (tkphd) [reply results/benchmark4h.1.yaml:6]: Curses! A perfect defense.

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1497.surge.sh

**Comment by pfhub** [comment]
Successful teardown of random-cat-1497.surge.sh

**Pull request merged**